src/evict/evict_conn.c

```diff
@@ -177,8 +177,20 @@ __evict_validate_config(WT_SESSION_IMPL *session, const char *cfg[])
 }
 
 /*
+ * NO_AUTO_FORMAT
+ *
  * __wt_evict_config --
- *     Configure or reconfigure eviction.
+ *     This function parses eviction-related configuration strings during `wiredtiger_open` or
```

Comment:
A lot of the new comments begin with "This function". It reads a bit nicer if we use the active tense instead:
 ```suggestion
  *     Parse eviction-related configuration strings during `wiredtiger_open` or
```
Can you review the comments to see where we can make this change?

Reply:
Updated in [a00febf](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/a00febf2327f469281978f90cd958a3713f91813)